pkg/controller/daemon/daemon_controller.go

```diff
@@ -842,6 +844,12 @@ func (dsc *DaemonSetsController) podsShouldBeOnNode(
 				// Emit an event so that it's discoverable to users.
 				dsc.eventRecorder.Eventf(ds, v1.EventTypeWarning, FailedDaemonPodReason, msg)
 				podsToDelete = append(podsToDelete, pod.Name)
+			} else if pod.Status.Phase == v1.PodSucceeded {
+				msg := fmt.Sprintf("Found succeeded daemon pod %s/%s on node %s, will try to delete it", pod.Namespace, pod.Name, node.Name)
+				logger.V(2).Info("Found succeeded daemon pod on node, will try to delete it", "pod", klog.KObj(pod), "node", klog.KObj(node))
+				// Emit an event so that it's discoverable to users.
+				dsc.eventRecorder.Eventf(ds, v1.EventTypeWarning, SucceededDaemonPodReason, msg)
```

Comment:
Rationale for the warning?  If every drain can generate this, and drain is “normal”, is it a warning?

Just want to understand the pattern we’re following here.

Reply:
yes, warning is too strong. I just followed the pattern for Failed pod. For now, I'm keeping the warning level for handling of Failed pods.

Reply:
forgot to commit?

Reply:
yes, got distructed. pushed now.